galaxy_api/api/exceptions.py

```diff
@@ -43,7 +45,20 @@ def _handle_drf_api_exception(exc):
 
 
 def _handle_api_client_exception(exc):
-    return HttpResponse(exc.body, status=exc.status, content_type=exc.headers['Content-Type'])
+    try:
```

Comment:
try..except block is too broad. Only `json.loads` can raise `ValueError` here.

Reply:
I think what you mean is that the try...excpet block should be around the line of code that may throw an error. Something more like the following:

```
try:
    error_list = json.loads(exc.body)
except ValueError:
    ...
```

Reply:
Agreed.